README.md

```diff
@@ -45,46 +48,45 @@ Move2 location object
 
-`Altitude Column` (character): The name of the column containing altitude data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty altitude column. 
+`Altitude Column` (character): Column name of the input data containing altitude data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty `altitude` column. 
 
-`Temperature Column` (character): The name of the column containing temperature data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty temp column. 
+`Temperature Column` (character): Column name of the input data containing temperature data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty `temperature` column. 
 
-`Heading Column` (character): The name of the column containing heading data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty heading column. 
+`Heading Column` (character): Column name of the input data containing heading data (if any). The column must already be contained within the input dataset. Leaving this empty generates an empty `heading` column. 
 
 `Keep Essential Columns` (logical): If TRUE, the output data contains only the following columns (but not all, depending on which settings above are used): *temperature*, *heading*, *altitude*, *import_marked_outlier*, *index*, *hour*, *min*, *secs*, *hourmin*, *yearmonthday*, *gap_mins*, *kmph*, *dist_m*, *x (UTM)*, *y (UTM)*, *geometry (sf)*, *lon*, *lat*, *study*.
 
 ### Most common errors
 
--   For any `column name` settings, please ensure:
+-   For any `column name` settings, please ensure that:
 
-    1.  That the named column is **contained within the input dataset**
-    2.  That the column name has been **spelled with complete accuracy**
+    1.  the named column is **contained within the input dataset**
+    2.  the column name has been **spelled with complete accuracy**
 
     or an error will be returned and processing will stop. Check the logs for the list of column names in the input dataset.
 
--   Any `column` settings can be bugged by a column name with two (and, sometimes, one) periods: for example, 'altitude.col.xyz' could not be recognized. To solve this bug, any column containing a period must be duplicated (instead of renamed) with the new name `altutide/temperature/heading`.
+-   Any `column` settings can be bugged by a column name with two (and, sometimes, one) periods: for example, 'altitude.col.xyz' could not be recognized. To solve this bug, any column containing a period must be duplicated (instead of renamed) with the new name `altitude/temperature/heading`.
```

Comment:
is this something the user has to do?  how would they go about changing a column name of something coming from moveapps?

Reply:
Yeah, I'm not sure about that statement either. My understanding is that the code is solving the issue automatically, i.e. it duplicates the column when it's name contains a period.  @callumjclarke, can you confirm please?